examples/backup/backup-include-ns.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: backup.kurator.dev/v1alpha1
+kind: Backup
+metadata:
+  name: include-ns
```

Comment:
need a more readable name since this will be presented in website

Reply:
ok